master/static/migrations/20231108114423_create-generic-tasks.tx.up.sql

```diff
@@ -0,0 +1,33 @@
+ALTER TYPE task_type RENAME TO _task_type;
```

Comment:
one of the failing linters is for the migration timestamps. to make it easier to fix and a bit more neat overall, I can suggest merging all migrations into one (i.e. single `up` + single `down` file)